.circleci/config.yml

```diff
@@ -423,103 +518,24 @@ jobs:
   # --------------------------------------------------
 
   unit_testing_php_56:
-    docker:
-      - image: circleci/php:5.6
-        auth:
-          username: $DOCKERHUB_USERNAME
-          password: $DOCKERHUB_PASSWORD
-      - image: circleci/mysql:5.7
-        auth:
-          username: $DOCKERHUB_USERNAME
-          password: $DOCKERHUB_PASSWORD
+    executor: php_56_mysql
     steps:
-      - checkout
-      - attach_workspace:
-          at: ~/project
-      - restore_composer_cache
-      - wait_for_mysql
-      - install_wordpress_testing_suite
-      - run_phpunit_singlesite_coverage_reports
-      - store_artifacts:
-          path: /tmp/artifacts/code-coverage/
-          destination: phpunit-coverage
-      - store_test_results:
-          path: /tmp/artifacts/code-coverage/
-      - run:
-          name: "Run PHPUnit - Multi Site"
-          command: WP_MULTISITE=1 composer run test
-          working_directory: /tmp/wordpress/wp-content/plugins/coblocks
+      - run_php_unit_tests
 
   unit_testing_php_73:
-    docker:
-      - image: circleci/php:7.3.23-apache-node
-        auth:
-          username: $DOCKERHUB_USERNAME
-          password: $DOCKERHUB_PASSWORD
-      - image: circleci/mysql:5.7
-        auth:
-          username: $DOCKERHUB_USERNAME
-          password: $DOCKERHUB_PASSWORD
+    executor: php_73_mysql
```

Comment:
you need mysql for unit tests?

Reply:
I just copied whatever was already in there. We can look into it further in another PR. This was just some cleanup.